plugins/nanospell/plugin.js

```diff
@@ -96,12 +96,16 @@
 				(block ? blockIsStartNode : true); // check we don't enter nested blocks (special list case since it's not considered a limit)
 
 			// If it's not a rootBlock text node, check to see if we hit a nested block element
+			// or another element that logically should cause a word break
+
 			if (!condition) {
-				if (isNotBookmark(node) &&
+				if (node.type == CKEDITOR.NODE_ELEMENT &&
+					isNotBookmark(node) &&
 				 	((blockLimit && !blockLimitIsStartNode) ||
-					(block && !blockIsStartNode))) {
+					(block && !blockIsStartNode) ||
+					node.is('br') || node.is('sup') || node.is('sub'))) {
```

Comment:
we can actually do

`node.is('br', 'sup', 'sub')`

at least this is what http://docs.ckeditor.com/#!/api/CKEDITOR.dom.element-method-is would indicate.